### PR TITLE
Move vscode and r-studio to the base layer

### PR DESCRIPTION
**Pull request by atheo89**
Move vs code, r-studio, and cuda r-studio to the base layer to be included as defaults

Ref Issue: https://github.com/opendatahub-io/notebooks/issues/283

## Description
This PR vs code, r-studio, and cuda r-studio to the base layer and removes them from the additional 

## How Has This Been Tested?
1. Deploy the operator by using the following config on the DataScienceCluster object
```
    workbenches:
      devFlags:
        manifests:
          - contextDir: components/odh-notebook-controller/config
            sourcePath: ''
            uri: 'https://github.com/opendatahub-io/kubeflow/tarball/v1.7.0-5'
          - contextDir: components/notebook-controller/config
            sourcePath: ''
            uri: 'https://github.com/opendatahub-io/kubeflow/tarball/v1.7.0-5'
          - contextDir: ''
            sourcePath: base
            uri: 'https://github.com/opendatahub-io/notebooks/tarball/pull/300/head'
      managementState: Managed
```
2. Ensure that the images are there
image

3. Spin up any of these three notebooks and check that they are working as expected. 

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: manifests/overlays/additional/kustomization.yaml [hunk 1]
```diff
@@ -1,13 +0,0 @@
----
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-
-resources:
-  - ../../base
-  - code-server-notebook-imagestream.yaml
-  - rstudio-notebook-imagestream.yaml
-  - rstudio-gpu-notebook-imagestream.yaml
```
Comment (harshad16) [inline manifests/overlays/additional/kustomization.yaml:9]: I would suggest, keep this file
and remove all these parts.

reasoning: in this way, anyone can use additional overlay for deployment and in future for other new inclusion
people would know where to add new yaml, instead of creating the whole kustomize.

File: manifests/overlays/additional/README.md [hunk 2]
```diff
@@ -1,11 +0,0 @@
...
-===
-
-All of the notebooks in this repo are supported by the ODH Community as a Tier 1 component but not part of the ODH Core deployment.
-The source build files for any notebook in this repo must exist under the [opendatahub-io](https://github.com/opendatahub-io) organization
-
-Notebooks:
-===
-* [Code Server Notebook ](https://github.com/opendatahub-io/notebooks/tree/main/codeserver) - Notebook image, allows to run Visual Studio Code (VSCode)
-* [R Studio Notebook](https://github.com/opendatahub-io/notebooks/tree/main/rstudio) - Notebook image, integrated development environment (IDE) for R
-* [CUDA R Studio Notebook](https://github.com/opendatahub-io/notebooks/tree/main/rstudio) - Notebook image, integrated development environment (IDE) for R with embedded CUDA capabilities
```
Comment (harshad16) [inline manifests/overlays/additional/README.md:11]: Please revert this change 
and only remove notebooks description from the bottom.

```
Notebooks:
===
- None
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
some suggestion

**Review by jstourac — APPROVED** [review]

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve
